src/components/LocationMarker/index.js

```diff
@@ -38,19 +38,30 @@ const LocationMarker = ({ geolocation }) => {
 
   const { accuracy, latitude, longitude } = geolocation
 
+  const removeLocationDot = () => {
+    locationDot.remove()
+  }
+
```

Comment:
This function can be moved out of the scope of the `LocationMarker` component

Reply:
✅